### PR TITLE
s3gw: drop obsolete UI build files, update UI dockerfile for release

### PR DESCRIPTION
**Pull request by jecluis**
We had a few files for building the UI that were not being used and could be considered obsolete.

The UI has also changed its build, and the Dockerfile has changed location. This patch set includes adjusting for the release, which means it needs to adjust the submodule's version as well.

Signed-off-by: Joao Eduardo Luis \<joao@suse.com>

### CODE REVIEW

File: ui
Comment (votdev) [inline ui:1]: Could you please update the ui submodule, i've done some cleanup with https://github.com/aquarist-labs/s3gw-ui/pull/258.

**Review by votdev — DISMISSED** [review]
LGTM

**Comment by jecluis** [comment]
@votdev updated the submodule. Needs a new approval.

**Review by votdev — APPROVED** [review]
LGTM

**Pull request merged**